### ISSUE TITLE
FR: Allow custom fonts

### ISSUE THREAD

**Issue by mturilin** [omnivore-app/omnivore#1816]
I want to use some beautiful fonts like Vollkorn. Would it be possible to enable custom fonts?

**Comment by jacksonh** [comment]
Hey, this is a cool idea. I think on the web at least it seems like it could be possible, or did you want this on iOS/Android?

**Comment by mturilin** [comment]
I use PWA on Android. Would it count as Web?

**Comment by jacksonh** [comment]
Yeah, sorry I forgot to reply. The PWA is web, so we should figure out how to do this across web and maybe iOS (not sure how possible it will be there)

**Comment by phdpe** [comment]
Hi, is there a plan to implement this for Android?

**Comment by Podginator** [comment]
Closing due to age, and the self-hosted nature of Omnivore. Done as a batch action. If this issue still exists, please reopen it.
